.github/workflows/codeql.yml

```diff
@@ -0,0 +1,34 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '31 6 * * 1'
```

Comment:
Why do we want to run this on a periodic basis? Should be sufficient to run on PRs and on merge events for master, no?

Reply:
ya i had that doubt, should we run this once in a day, week or only during PRs and on merge, so should i remove the cron?